drydock_provisioner/control/base.py

```diff
@@ -11,18 +11,36 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import falcon.request as request
 import uuid
```

Comment:
I think this is an outdated change, I cleaned up the imports in the latest commits.